tool/world_options.re

```diff
@@ -0,0 +1,300 @@
+Tas::set_reflection_render_scale(SETTINGS.reflection_render_scale);
+Tas::set_lighting_casts_shadows(SETTINGS.lighting_casts_shadows);
+Tas::enable_disable_sky_light(SETTINGS.sky_light_enabled);
+Tas::set_time_dilation(SETTINGS.time_dilation);
+Tas::set_gravity(SETTINGS.gravity);
+Tas::set_time_speed(SETTINGS.sky_time_speed);
+Tas::set_sky_light_brightness(SETTINGS.sky_light_brightness);
+Tas::set_sky_light_intensity(SETTINGS.sky_light_intensity);
+Tas::set_stars_brightness(SETTINGS.stars_brightness);
+Tas::set_fog_enabled(SETTINGS.fog_enabled);
+Tas::set_cloud_speed(SETTINGS.cloud_speed);
+Tas::set_gamma(SETTINGS.display_gamma);
```

Comment:
You forgot
* `set_kill_z`
* `set_cloud_color`
* `set_outro_dilated_duration`
* `set_outro_time_dilation`
* `set_sun_color`